impl/test/src/test/resources/workflows-samples/set-listen-to-any.yaml

```diff
@@ -10,4 +10,6 @@ do:
   - callDoctor:
       listen:
         to:
-          any: []
```

Comment:
This test was intenationally specyfing[] to test register all, it should not be associated with an specific type